### PR TITLE
Edit pets and tasks

### PR DESCRIPTION
**Pull request by nikkypyra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,50 @@
...
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+<div align="center">
```
Comment (ghost) [inline README.md:1]: Oh, didn't even think of changing this! Like it and will do it too.

File: src/App.js [hunk 2]
```diff
@@ -69,6 +71,18 @@ export default function App() {
                   />
                 )}
               </Route>
+              <Route path="/pet/:id/update-pet">
+                {user.id ? (
+                  <EditPetForm pets={pets} deletePet={deletePet} />
```
Comment (ghost) [inline src/App.js:76]: Some how deletePet doesn't feel right ;) Feels like giving it to the shelter ... Just kidding.

File: src/pages/EditPetForm.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+          })
+      }
+    )
+  }
+}
+
+const Form = styled.form`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto auto auto auto auto;
```
Comment (ghost) [inline src/pages/EditPetForm.js]: Not sure, but wouldn't `grid-template-rows: auto;` suffice?

File: src/pages/EditPetForm.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  align-items: center;
+  color: var(--secondary);
+  margin: 20px;
+  padding: 12px;
+  border: 4px solid var(--tertiary);
+  border-radius: 12px;
+  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.16);
```
Comment (ghost) [inline src/pages/EditPetForm.js]: Anywhere else you are not using rgba, maybe do it consistently? And thinking about it: A may actually be doing the same thing ...

**Review by ghost — APPROVED** [review]
Looks very good and also thorough to me liebe Nikky!

File: src/App.js (hunk 2 above)
Comment (nikkypyra) [reply src/App.js:76]: I know, right! I didn't really want to build that feature, but I guess we have to be practical :(

File: src/pages/EditPetForm.js (hunk 3 above)
Comment (nikkypyra) [reply src/pages/EditPetForm.js]: You are probably right, I will change that :)

File: src/pages/EditPetForm.js (hunk 4 above)
Comment (nikkypyra) [reply src/pages/EditPetForm.js]: That's a good point!

**Pull request merged**